cdc/Makefile

```diff
@@ -43,8 +43,8 @@ FAILPOINT_DISABLE := $$(echo $(FAILPOINT_DIR) | xargs $(FAILPOINT) disable >/dev
 
 RELEASE_VERSION =
 ifeq ($(RELEASE_VERSION),)
-	RELEASE_VERSION := v5.4.0-master
-	release_version_regex := ^v5\..*$$
+	RELEASE_VERSION := v1.0.0-master
```

Comment:
We may need check the tikv version and apiversion to make sure the compatibilcity.

Reply:
Agree. I open an issue https://github.com/tikv/migration/issues/170 for this.